app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+    def new
+        @comment = Comment.new
+    end
+
+    def index
+        #@posts = Post.all
+    end
+
+    def show
+        @current_post = Post.find(params[:id])
+    end
+
+    def edit
+        @comment = Comment.find(params[:id])
+    end
+
+    def update
+        @comment = Comment.find(params[:id])
+        if @comment.update_attributes(comment_params)
+          flash[:success] = "Comment updated"
+          redirect_to @post
+        else
+          render 'edit'
+        end
+    end
+
+    def create
+        @comment = Comment.new(comment_params)
+        @comment.user_id = 1
+        @comment.points = 0
+        if @pcommentost.save!
```

Comment:
Note the difference between .save and .save!

http://api.rubyonrails.org/classes/ActiveRecord/Validations.html#method-i-save-21

With the method you're currently using, is it possible to ever reach your else statement?